util/settings.ts

```diff
@@ -90,3 +90,25 @@ export const putSettings =
 		)
 		return { name: Name }
 	}
+
+export const deleteSettings =
+	({
+		ssm,
+		stackName,
+		scope,
+		system,
+	}: {
+		ssm: SSMClient
+		stackName: string
+		scope: Scopes
+		system: Systems
+	}) =>
+	async ({ property }: { property: string }): Promise<{ name: string }> => {
+		const Name = `/${stackName}/${scope}/${system}/${property}`
```

Comment:
**nitpick:** it would be a good idea to extract this into a shared method so that `putSettings` and `deleteSettings` use the same name.